docs/journalist.rst

```diff
@@ -645,6 +645,53 @@ audio, and begin publishing important, high-impact work!
          <getting_the_most_out_of_securedrop>` to read about
          encouraging sources to use SecureDrop.
 
+Deleting submissions and source accounts
```

Comment:
FWIW, I decided to add this to the end of the guide since it's impossible to make a judgment about what submissions should be deleted until after review on the SVS, and inserting it earlier might be confusing for someone trying to get a handle on the whole workflow.